.github-actions-pre-commit-config.yaml

```diff
@@ -0,0 +1,113 @@
+---
+default_language_version:
+  # force all unspecified python hooks to run python3
+  python: python3
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.5.0
+    hooks:
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-merge-conflict
+      - id: check-xml
+      - id: debug-statements
+      - id: detect-aws-credentials
+        args:
+          - --allow-missing-credentials
+      - id: detect-private-key
+      - id: end-of-file-fixer
+        exclude: files/(issue|motd)
+      - id: mixed-line-ending
+        args:
+          - --fix=lf
+      - id: pretty-format-json
+        args:
+          - --autofix
+      - id: requirements-txt-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.22.0
+    hooks:
+      - id: markdownlint
+        args:
+          - --config=.mdl_config.json
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.21.0
+    hooks:
+      - id: yamllint
+  - repo: https://github.com/detailyang/pre-commit-shell
+    rev: 1.0.5
+    hooks:
+      - id: shell-lint
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.9
+    hooks:
+      - id: flake8
+        additional_dependencies:
+          - flake8-docstrings
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.1.0
+    hooks:
+      - id: pyupgrade
+  # Run bandit on "tests" tree with a configuration
+  - repo: https://github.com/PyCQA/bandit
+    rev: 1.6.2
+    hooks:
+      - id: bandit
+        name: bandit (tests tree)
+        files: tests
+        args:
+          - --config=.bandit.yml
+  # Run bandit everything but tests directory
+  - repo: https://github.com/PyCQA/bandit
+    rev: 1.6.2
+    hooks:
+      - id: bandit
+        name: bandit (everything else)
+        exclude: tests
+  - repo: https://github.com/python/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+  - repo: https://github.com/asottile/seed-isort-config
+    rev: v2.1.0
+    hooks:
+      - id: seed-isort-config
+  - repo: https://github.com/pre-commit/mirrors-isort
+    # pick the isort version you'd like to use from
+    # https://github.com/pre-commit/mirrors-isort/releases
+    rev: v4.3.21
+    hooks:
+      - id: isort
+  - repo: https://github.com/ansible/ansible-lint.git
+    rev: v4.2.0
+    hooks:
+      - id: ansible-lint
+      # files: molecule/default/playbook.yml
+  - repo: https://github.com/antonbabenko/pre-commit-terraform.git
+    rev: v1.29.0
+    hooks:
+      - id: terraform_fmt
+      # I am commenting out this hook because terraform validate
```

Comment:
I was thinking about the way we are doing this just this morning.  With separate files, they will eventually fall out of sync with each other.  I think what makes more sense is to do the following in the GitHub Actions workflow:
1. Run `sed` or something similar against the default `.pre-commit-config.yaml` file to comment out the `terraform_validate` line and generate a new file `/tmp/.github-actions-pre-commit-config.yaml`.
2. Run `pre-commit` with the `/tmp/.github-actions-pre-commit-config.yaml` configuration.

Reply:
Ah, that is a very good idea!

Reply:
Fantastic idea! Implemented in 022fdf0 (and fixed in 0724b04).

Reply:
Excellent work @mcdonnnj!